src/main/java/Quiz/dto/QuestionIdDto.java

```diff
@@ -0,0 +1,45 @@
+package Quiz.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Collection;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class QuestionIdDto {
+    private int id;
+    private String description;
+    private Collection<QuestionOptionIdDto> options;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public Collection<QuestionOptionIdDto> getOptions() {
+        return options;
+    }
+
+    public void setOptions(Collection<QuestionOptionIdDto> options) {
+        this.options = options;
+    }
```

Comment:
Please use project lombok so we can be free of basic getters and setters in Moldes and DTOs

    <!-- Object mapper-->
    <dependency>
        <groupId>org.projectlombok</groupId>
        <artifactId>lombok</artifactId>
        <version>1.18.10</version>
    </dependency>

Reply:
I have now added Lombok and removed all getters and setters. For now, I have only used the @Data annotation from Lombok, but perhaps we should use more detalied annotaions such as @Getter and @Setter.